app/Models/User.php

```diff
@@ -51,13 +54,24 @@ class User extends Authenticatable
 
     #region Relationships
 
-    function group_memberships()
+    public function groups()
     {
-        return $this->hasMany(GroupMember::class);
+        return $this->hasManyThrough(Group::class, GroupMember::class, 'user_id', 'id', 'id', 'group_id');
```

Comment:
What? No. 

Group->Workspace->Topic

Topic has one Group. The Group Members all have one Group. It's the same Group. You've made the wrong relation type through the wrong object, and now just have a collection of Groups that are all literally the same Group for no reason.

Reply:
Fixed!